ssh-krb-node-executor-plugin/contents/scp-krb-executor.sh

```diff
@@ -13,36 +13,49 @@ shift
 SRC_FILE=$1
 shift
 DEST_FILE=$1
-KERB_KEYTAB=$RD_CONFIG_KERBEROS_KEYTAB
 
-if [ ! -f "$KERB_KEYTAB" ]; then
-  echo "Keytab $KERB_KEYTAB not found" >&2
+KRB_USER=$RD_CONFIG_KERBEROS_USER
```

Comment:
May I ask why you're (re)defining these local variables instead of using the configuration options that Rundeck injects (`$RD_CONFIG_WHATEVER`) directly?

Reply:
I understand that shortening the variable names might increase the readability but on the other hand it adds more moving parts that could introduce regressions. As this software does not have any test, the shorter the diff, the better, so please do use the Rundeck-injected variables if the values do not require to be transformed in any way.